content/src/roadmaps/license-tasks/consulting-firm-headhunter-reg.md

```diff
@@ -1,13 +1,13 @@
 ---
-id: "undefined"
+id: "consulting-firm-headhunter-reg"
 webflowId: "65bd67f3f9dc39f100108f8b"
 urlSlug: "consulting-firm-headhunter-reg"
 name: "Register Your Consulting Firm (Headhunter) "
 webflowName: "Employment and Personnel Services: Consulting Firm (Headhunter)"
 filename: "consulting-firm-headhunter-reg"
 callToActionLink: "https://www.njconsumeraffairs.gov/regulated/Pages/Regulated-Business-Online-Registration.aspx"
 callToActionText: "Register My Consulting Firm"
-industryId: "employment-agency"
+industryId: "empl**oyment-agency**"
```

Comment:
Are these asterisks intentional? If so, can you clarify the purpose? Is this to help the triage?